test/cmd/lfstest-gitserver.go

```diff
@@ -481,6 +497,98 @@ func storageHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 		w.WriteHeader(404)
+	case "HEAD":
+		// tus.io
+		if len(r.Header.Get("Tus-Resumable")) == 0 {
+			log.Fatal("Missing Tus-Resumable header in request")
+			w.WriteHeader(400)
+			return
+		}
+		parts := strings.Split(r.URL.Path, "/")
+		oid := parts[len(parts)-1]
+		if by, ok := largeObjects.GetIncomplete(repo, oid); ok {
```

Comment:
As another note, [`strconv.Atoi`](https://golang.org/pkg/strconv/#Itoa) is shorthand for the `FormatInt` call, so you can save some characters.